azizmashkour/metrics-api
# config and start implementing tests

## Description

This PR implements and configures the test environment for our Rails API. It includes the test for the following:
- [x] the metrics model validation
- [x] the metrics API ~ `GET` and `POST` methods
- [x] the metrics serializer

## How to test
1. Make sure you run bundle to install the gems
2. run the command `rspec ${with-the-path-to-the-spec-to-be-tested}`. Eg: rspec spec/requests/get_metrics_spec.rb
